src/scalar/mod.rs

```diff
@@ -14,6 +19,8 @@ lazy_static! {
         let mut hm: HashMap<String, Arc<dyn Scalar + Send + Sync>> = HashMap::new();
         hm.insert("Email".to_string(), Arc::new(Email::default()));
         hm.insert("PhoneNumber".to_string(), Arc::new(PhoneNumber::default()));
+        hm.insert("Date".to_string(), Arc::new(Date::default()));
+        hm.insert("Url".to_string(), Arc::new(Url::default()));
         hm
```

Comment:
In the current implementation, with every scalar, we need to update `CustomScalar` and `CUSTOM_SCALARS`. This is a bit redundant, can we do something so that we update only at one place and the other is auto generated?